UI/window-basic-settings.cpp

```diff
@@ -5267,12 +5290,30 @@ void OBSBasicSettings::SimpleReplayBufferChanged()
 	bool replayBufferEnabled = ui->simpleReplayBuf->isChecked();
 	bool lossless = qual == "Lossless";
 	bool streamQuality = qual == "Stream";
-
+	int abitrate = 0;
```

Comment:
Re-add the blank line below this newly added line.

Reply:
done